lib/src/io.cpp

```diff
@@ -494,7 +495,19 @@ noexcept (false) {
         dl::obname tmp{ dl::origin{ origin },
                         dl::ushort{ copy },
                         dl::ident{ std::string{ id, id + idlen } } };
-        xs.emplace_back(tmp.fingerprint("FRAME"), iflr.tell);
+        std::string fp = tmp.fingerprint("FRAME");
+
+        /* Although index.size() often is fairly large, the number of
+         * unique frames are typically just an handful. Hence the repeated
```

Comment:
number of unique frames (is) typically (small)
or
there are just (a) handful of unique frames?

number is a handful sounds strange to me